README.md

```diff
@@ -57,21 +57,80 @@ npm update @unstoppabledomains/resolution --save
 
 ## Using Resolution
 
-Create a new project.
+## Initialize with Unstoppable Domains' Proxy Provider
 
-```shell
-mkdir resolution && cd $_
-yarn init -y
-yarn add @unstoppabledomains/resolution
+```javascript
+const {default: Resolution} = require('@unstoppabledomains/resolution');
+
+// obtain a key from https://unstoppabledomains.com/partner-api-dashboard if you are a partner
+const resolution = new Resolution({ apiKey: "<api_key>" });
```

Comment:
We should call out that the `apiKey` is only for ETH and MATIC, ZIL resolution will still require users to bring their own keys.

Reply:
updated!